src/main/java/com/denizenscript/ddiscordbot/DenizenDiscordBot.java

```diff
@@ -81,6 +81,7 @@ public void onEnable() {
             ScriptEvent.registerScriptEvent(DiscordUserLeavesScriptEvent.class);
             ScriptEvent.registerScriptEvent(DiscordUserNicknameChangeScriptEvent.class);
             ScriptEvent.registerScriptEvent(DiscordUserRoleChangeScriptEvent.class);
+            ScriptEvent.registerScriptEvent(DiscordInviteCreateScriptEvent.class);

```

Comment:
the registration list is in alphabetical order (ie the order that your intellij file list should have them in anyway)